### PR TITLE
Updated README to include corev-dv extension descriptions

### PR DESCRIPTION
**Pull request by silabs-hfegran**
See issue #1068 

Signed-off-by: Henrik Fegran <Henrik.Fegran@silabs.com>

### CODE REVIEW

**Comment by MikeOpenHWGroup** [comment]
This is fantastic stuff @silabs-hfegran, thanks for doing this.   Of course, I have a couple of pedantic comments.

File: cv32e40p/env/corev-dv/README.md [hunk 1]
```diff
@@ -10,3 +10,164 @@ TODO: re-direct the generated assembly language tests-programs to a central loca
...
+
+#### cv32e40p\_debug\_rom\_gen
+Description:<br>
+Enables core-v-verif custom debug rom, with config fetched from corev configuration object.
+
+| Extends               |
+|:----------------------|
+| `riscv_debug_rom_gen` |
+
+| Implements                    | Purpose |
```
[the remark this answers is not in the document]
Comment (silabs-hfegran) [reply cv32e40p/env/corev-dv/README.md:46]: Correct, spanning does not work (at least not in a way I could find, so this was to me the next best solution)
That is why I decided to have the first column empty for the next two lines, as a way to emulate this behavior.

I would not mind changing it if you have a better suggestion.

This is how the output is rendered:
image

[the remark this answers is not in the document]
Comment (MikeOpenHWGroup) [reply cv32e40p/env/corev-dv/README.md:46]: AFAIK, that is the best we can get from [GFM](https://github.github.com/gfm/) (yes, its a thing).  So let's go with that.

**Review by MikeOpenHWGroup — APPROVED** [review]

**Comment by MikeOpenHWGroup** [comment]
Thanks @silabs-hfegran!

**Pull request merged**